src/main/java/org/terasology/climateConditions/AffectBodyTemperatureEvent.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.climateConditions;
+
+import org.terasology.entitySystem.event.AbstractValueModifiableEvent;
+
+/**
+ * This event is sent out by the {@link BodyTemperatureSystem} to allow for other systems to modify the change in body
+ * temperature.
+ */
+public class AffectBodyTemperatureEvent extends AbstractValueModifiableEvent {
+    private boolean isNegative = false;
```

Comment:
Why do we need to keep this state? Can't we resolve this property directly from the modifiable event itself?

Reply:
As commented further below, maybe we can hide this inside the event? Would it make sense to override `getResultValue` to multiply with `-1` if the value should be negative?